samples/drivers/espi/src/main.c

```diff
@@ -0,0 +1,339 @@
+/*
+ * Copyright (c) 2012-2014 Wind River Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <zephyr.h>
+#include <misc/printk.h>
+#include <device.h>
+#include <soc.h>
+
+#include <gpio.h>
+#include <espi.h>
+
+#define SLPS5_DLY               60000
+#define SLPS4_DLY               60000
+#define SLPS3_DLY               30000
+
+/* Board specific definitions */
+#ifdef CONFIG_ESPI_XEC
```

Comment:
I guess it would be better to just have a generic macro name. See what's been done for samples/driver/watchdog for instance. The generic name comes from soc's dts_fixup.h